meta_test.go

```diff
@@ -515,6 +543,38 @@ func TestSetMeta_object_with_array(t *testing.T) {
 	}
 }
 
+func TestSetMeta_json_object(t *testing.T) {
```

Comment:
Do we need an error test case?

Reply:
If you follow up by adding a case, can you @mention me so I can see it?